.travis.yml

```diff
@@ -258,6 +258,13 @@ jobs:
                   echo "REPORTED SUCCESSFULLY";
                 fi
 
+        - if: NOT tag =~ ^RELEASE_CANDIDATE
```

Comment:
@Artem-applitools don't you mind if we would run this tutorial tests as part of weekly regression? For this thing you need to add: `if: NOT tag =~ ^RELEASE_CANDIDATE AND type = cron`